inc/template-functions.php

```diff
@@ -97,3 +97,21 @@ function interconnection_cap_description( $description ) : string {
 	return $description;
 }
 add_filter( 'get_the_archive_description', 'interconnection_cap_description' );
+
+
+/**
+ * Toggles Polylang Content duplication for the current user.
+ *
+ * This function updates the 'pll_duplicate_content' user meta to control content duplication
+ * across languages in Polylang. The feature is controlled by the 'pll-duplicate' option,
+ * and when enabled, it allows the content to be duplicated across different languages.
+ *
+ * It makes the link https://<environment>/wp-admin/post-new.php?post_type=post&from_post=<POST-ID>&new_lang=<LANG>&_wpnonce=<NONCE>
+ * to create a new post in the language specified by the 'new_lang' parameter.
+ *
+ * See #879 Diff bug ticket for more details.
+ */
+function toggle_polylang_content_duplication() {
+    update_user_meta( get_current_user_id(), 'pll_duplicate_content', [ 'post' => true ] );
```

Comment:
Running this on `init` will cause a user meta update call on every logged-in page view. It would be better to run this on the `wp_login` action: https://developer.wordpress.org/reference/hooks/wp_login/

The arguments passed to the `wp_login` action are
```php
string $user_login, WP_User $user
```
so you can get the user ID off of the `$user` argument instead of checking `get_current_user_id()`.